### PR TITLE
feat: run standup reminder via @artsy/cli every Monday at 9am ET

### PR DESCRIPTION
**Pull request by dblandin**
Use the Artsy CLI and scheduled GitHub actions for our Monday Informationals.

Currently in-progress. This doesn't support all of the Peril-backed tasks yet.

- [ ] Closed Source Rationale Check
- [ ] Compare Schemas
- [x] Standup Reminder
- [x] Weekly RFC Summary

#FF #FutureFriday

**Example**

Screen Shot 2021-02-07 at 3 33 17 PM

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Found a new file with underscores, this repo uses kebab-case for files: .github/workflows/monday_informationals.yml
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 501d40f3a071124ae30ef735440511034e8faa55
</p>

File: .github/workflows/mondayInformationals.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  schedule:
+    - cron: "0 14 * * MON"
+
+jobs:
+  monday_informationals:
+    name: Monday Informationals
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Dependencies
+        run: yarn global add --silent @artsy/cli@alpha
```
Comment (damassi) [inline .github/workflows/mondayInformationals.yml:12]: Don't think it makes a big difference, but could we [use `npx` instead](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) (which ships with node) and skip the install step?

Comment (dblandin) [reply .github/workflows/mondayInformationals.yml:12]: Ahh, that's a good idea. What if we were invoking the CLI multiple times within the same workflow? Would using npx install it N times?

Comment (damassi) [reply .github/workflows/mondayInformationals.yml:12]: Yup, it would, so this would only be useful for one-off commands

**Comment by dblandin** [comment]
Going to merge this to test against the `@artsy/cli@alpha` version.

File: .github/workflows/mondayInformationals.yml (hunk 1 above)
Comment (jonallured) [reply .github/workflows/mondayInformationals.yml:12]: Would be cool to generate a docker image with the cli already installed so this is moot ya know?

**Pull request merged**